docs/notebooks/pulse-efficient-transpilation.ipynb

```diff
@@ -66,54 +107,67 @@
+    "\n",
+    "ecr_backend = service.least_busy(simulator=False, operational=True, filters=lambda b: b.configuration().processor_type['family'] == 'Eagle' and ('ecr' in b.basis_gates))\n",
+    "\n",
+    "ecr_backend"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "When running a generic quantum circuit on the hardware, the circuits need to be transpired into a basic set of gates so that it is compatible with a target backend. This process is handled by the [`qiskit.transpiler`](https://qiskit.org/documentation/apidoc/transpiler.html), and in the how-to guide, we will introduce two ways of defining the basis gate set."
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 4,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "# standard way: basis gate set that includes ECR gate\n",
-    "BASIS_GATES = ['ecr', 'x', 'sx', 'rz']\n",
+    "When running a generic quantum circuit on hardware, the circuits must be transpiled into a basic set of gates that are compatible with a target backend. This process is handled by the [`qiskit.transpiler`](https://qiskit.org/documentation/apidoc/transpiler.html).  This tutorial introduces two ways of defining the basis gate set:\n",
     "\n",
-    "# pulse efficient way: basis gate set that includes the pulse RZX gate\n",
-    "BASIS_GATES = ['rzx', 'x', 'sx', 'rz']"
+    "- Standard method: Basis gate set that includes ECR gate\n",
+    "    `BASIS_GATES = ['ecr', 'x', 'sx', 'rz']`\n",
+    "\n",
+    "- Pulse-efficient method: Basis gate set that includes the pulse RZX gate\n",
+    "    `BASIS_GATES = ['rzx', 'x', 'sx', 'rz']`"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "A standard way to transpile is to transpile into a basis gate set that includes a two-qubit entangling gate. Depending on the backend, the two-qubit entangling gate is either a `CXGate` or an `ECRGate`, which are pre-calibrated on the hardware. In practice, the two-qubit entangling gates have relatively longer gate times compared with single-qubit gates and are one of the main sources of errors. In this guide, we show a method to transpile into a `RZXGate` pulse gate. It is a continuous two-qubit entangling gate with the angle $\\theta$ as a gate parameter and has adjustable gate time accordingly. In some use cases, we show that the transpiled circuits are pulse-efficient, meaning they have fewer two-qubit gates and a shorter circuit duration. This is beneficial since the qubits have a finite coherence time. \n",
+    "The standard method is to transpile into a basis gate set that includes a two-qubit entangling gate. Depending on the backend, the two-qubit entangling gate is either a `CXGate` or an `ECRGate`, which are pre-calibrated on the hardware. In practice, two-qubit entangling gates typically have longer gate times than single-qubit gates, and are a main error source. \n",
```

Comment:
this information about transpilation feels a bit out of place, is there a reason it needs to be introduced this early and not in the transpilation section later on?